src/csharp/Pester/Factory.cs

```diff
@@ -42,16 +42,16 @@ public static ErrorRecord CreateShouldErrorRecord(string message, string file, s
         public static ErrorRecord CreateErrorRecord(string errorId, string message, string file, string line, string lineText, bool terminating)
         {
             var exception = new Exception(message);
-            var errorCategory = ErrorCategory.InvalidResult;
             // we use ErrorRecord.TargetObject to pass structured information about the error to a reporting system.
-            var targetObject = new Dictionary<string, object> { };
-            targetObject.Add("Message", message);
-            targetObject.Add("File", file);
-            targetObject.Add("Line", line);
-            targetObject.Add("LineText", lineText);
-            targetObject.Add("Terminating", terminating);
-            var errorRecord = new ErrorRecord(exception, errorId, errorCategory, targetObject);
-            return errorRecord;
+            var targetObject = new Dictionary<string, object>
+            {
+                { "Message", message },
+                { "File", file },
+                { "Line", line },
+                { "LineText", lineText },
+                { "Terminating", terminating }
```

Comment:
```suggestion
                ["Message"] = message,
                ["File"] = file,
                ["Line"] = line,
                ["LineText"] = lineText,
                ["Terminating"] = terminating,
```

I find this syntax little more obvious.